src/applications/applications.controller.ts

```
@@ -70,4 +71,22 @@ export class ApplicationsController {
   ) {
     return this.applicationsService.updateStatus(Number(id), updateStatusDto);
   }
+
+  @Get('/reports/csvexport')
+  @ApiOperation({ summary: 'Export applications as CSV', description: 'Exports applications for a given benefitId and report type as a CSV file.' })
+  @ApiBody({ type: CsvExportApplicationsDto })
+  @ApiResponse({ status: 200, description: 'CSV file with applications data', schema: { type: 'string', format: 'binary' } })
+  @ApiResponse({ status: 400, description: 'Missing or invalid parameters' })
+  async csvexport(@Query() dto: CsvExportApplicationsDto, @Res() res: Response) {
```

ESLint

[error] Replace `@Query()·dto:·CsvExportApplicationsDto,·@Res()·res:·Response` with `⏎····@Query()·dto:·CsvExportApplicationsDto,⏎····@Res()·res:·Response,⏎··`

(prettier/prettier)